mkat_fpga_tests/test_cbf.py

```diff
@@ -662,12 +662,17 @@ def test_sensor_values(self):
                 , 'Check the number of sensors in the list is equal to the '
                     'list of values received for {}'.format(roach.host))
 
-            for sensor in sensors_values[1:]:
-                sensor_name, sensor_status, sensor_value = sensor.arguments[2:]
-                # Check is sensor status is a Fail
-                Aqf.is_false((sensor_status == 'fail'),
-                    'Roach {}, Sensor name: {}, status: {}'
-                        .format(roach.host, sensor_name, sensor_status))
+            def roach_sensors_status(run_test=False):
```

Comment:
Should this not be a new test? I.e a method of the test class called test_roach_sensors or summin?